src/server/generic_family.cc

```diff
@@ -1495,6 +1536,7 @@ void GenericFamily::Register(CommandRegistry* registry) {
       << CI{"SCAN", CO::READONLY | CO::FAST | CO::LOADING, -2, 0, 0, 0, acl::kScan}.HFUNC(Scan)
       << CI{"TTL", CO::READONLY | CO::FAST, 2, 1, 1, 1, acl::kTTL}.HFUNC(Ttl)
       << CI{"PTTL", CO::READONLY | CO::FAST, 2, 1, 1, 1, acl::kPTTL}.HFUNC(Pttl)
+      << CI{"FIELDTTL", CO::READONLY | CO::FAST, 3, 1, 1, 1, acl::kFieldTtl}.HFUNC(FieldTtl)
```

Comment:
Maybe SetTTL would be a better name? Field doesn't indicate what type it really belongs to

Reply:
that's the whole point. I am going to add transparent support for hash family via the same command